### ISSUE TITLE
What's the point....

### ISSUE THREAD

**Issue by Mrtrex525** [TinyModularThings/Chunk-Pregenerator-Issue-Tracker#28]
I cant even use the Chunk Pre generator without paying like bro how greedy do they have to be to want monthly money for one fucking mod that only pre generator like wtf dude.

**Comment by Speiger** [comment]
@Mrtrex525 What are you talking about?
The "Pregenerator" part and a lot of the features (roughly 80%) of the mod is freely accessible and usable.
Or did you reduce Chunk Pregenerator down to just the preview which just suplemented 2-3 other features of the mod that is still accessible?

About why? Because I invested a lot of time, got nothing back and when I was in a situation where i had to ask for help and was left alone (and yet got requests for please being on the latest version).
Providing incentives were ignored too,
so i said: Ok I am trying this last ditch effort to keep the mod alive (Because yes the last 2-3 patches wouldn't have existed thanks to this) and went for it. With a Warning beforehand.

TL:DR I wanted to continue to provide a great mod, but couldn't due to limitations and other ways weren't viable and would have forced me to give up the mod (based on how much time it ate) so i had to go that route.

What about this is exactly greedy now?
I am a human being that provided a great tool for free to you guys (which is in the free version still better then a lot other tools)
and wanted to continue when i was at a point to either give it up or make it donationware.
Asking nicely was ignored, this worked.
If this would have failed I would have moved on. But it didn't
